src/components/app/details/appDetails/appDetails.service.ts

```diff
@@ -16,8 +16,11 @@ export function isDatasourceHealthy(datasourceId: number | string) {
     return fetchWithFullRoute(URL, 'GET');
 }
 
-export function getDeploymentStatusDetail(appId: string, envId: string, triggerId?: string): Promise<DeploymentStatusDetailsResponse> {
-  return get(`${Routes.DEPLOYMENT_STATUS}/${appId}/${envId}${triggerId ? `?wfrId=${triggerId}` : ``}`)
+export function getDeploymentStatusDetail(appId: string, envId: string, triggerId?: string, isHelm?: boolean): Promise<DeploymentStatusDetailsResponse> {
+
+    return get(`${isHelm ? Routes.HELM_DEPLOYMENT_STATUS : Routes.DEPLOYMENT_STATUS}/${appId}/${envId}${triggerId ? `?wfrId=${triggerId}` : ``}`)
```

Comment:
Extract `isHelm ? Routes.HELM_DEPLOYMENT_STATUS : Routes.DEPLOYMENT_STATUS` to solve sonar issue

Reply:
Done